modules/ossm-rn-known-issues.adoc

```diff
@@ -25,6 +25,7 @@ These limitations exist in {SMProductName}:
 
 * WebAssembly extensions are not supported on IBM Z and IBM Power Systems.
 
+* LuaJIT is not supported on IBM Z and IBM Power Systems.
```

Comment:
LuaJIT is supported on IBM Z also IBM Power Systems has been rebranded to IBM Power, replace with: 

* LuaJIT is not supported on IBM Power.

Reply:
Thanks, @SNiemann15! I've made this change.